blaze/expr/strings.py

```diff
@@ -83,6 +113,86 @@ class str_lower(UnaryStringFunction):
     def schema(self):
         return self._child.schema
 
+class str_isalnum(UnaryStringFunction): schema = bool_
+class str_isalpha(UnaryStringFunction): schema = bool_
+class str_isdecimal(UnaryStringFunction): schema = bool_
+class str_isdigit(UnaryStringFunction): schema = bool_
+class str_islower(UnaryStringFunction): schema = bool_
+class str_isnumeric(UnaryStringFunction): schema = bool_
+class str_isspace(UnaryStringFunction): schema = bool_
+class str_istitle(UnaryStringFunction): schema = bool_
+class str_isupper(UnaryStringFunction): schema = bool_
+
+class StrFind(ElemWise):
+    """
+    Find literal substring in string column.
+
+    """
+
+    _arguments = '_child', 'sub'
+    schema = datashape.Option(datashape.int64)
+
+
+@copydoc(StrFind)
+def str_find(col, sub):
+    if not isinstance(sub, basestring):
+        raise TypeError("'sub' argument must be a String")
+    return StrFind(col, sub)
+
+class Replace(ElemWise):
+    _arguments = '_child', 'old', 'new', 'max'
+    schema = datashape.Option(datashape.string)
+
+def str_replace(col, old, new, max=None):
+    _validate(old, 'old', basestring, 'string')
+    _validate(new, 'new', basestring, 'string')
+    _validate_optional(max, 'max', int, 'integer')
+    return Replace(col, old, new, max)
+
+class Pad(ElemWise):
+    _arguments = '_child', 'width', 'side', 'fillchar'
+    schema = datashape.Option(datashape.string)
+
+def str_pad(col, width, side=None, fillchar=None):
+    _validate(width, 'width', int, 'integer')
+    if side not in (None, 'left', 'right'):
+        raise TypeError('"side" argument must be either "left" or "right"')
+    _validate_optional(fillchar, 'fillchar', basestring, 'string')
+    return Pad(col, width, side, fillchar)
+
+class str_capitalize(UnaryStringFunction):
+    schema = datashape.Option(datashape.string)
```

Comment:
These schema should also be defined as properties and pass through `_child.schema`, I think.